site/web/assets/js/livestamp.js

```diff
@@ -1,5 +1,12 @@
 // Livestamp.js / v2.0.0 / (c) 2015 Matt Bradley / MIT License
```

Comment:
Can't we install this with yarn?

Reply:
Since this file contains `livestamp` and `moment` with other functions, I think we should rename it to something else. What about `base.js`? As it is included in base.html.twig?

Reply:
How does that answer the question I raised? Why can't we install livestamp through yarn?